src/documentation/basic-opts.page

```diff
@@ -30,21 +30,23 @@ Another possible way of sending this information to uniqush: the app send the in
 
 As mentioned in [basic concepts], a subscriber may have more than one delivery point subscribes a service. So several delivery points could subscribe same service under same subscriber. This requires the server side have some mechanism to authenticate the subscriber's identity.
 
-## Push Data to Subscriber
+## Sending Push Notifications to a Subscriber
 
-Once a service has more than one push service provider and more than one subscriber, uniqush can push data on behalf of that service to the apps running on subscribers' devices. Server side only need to provide service name, subscriber name and data. Uniqush will push the data to all active delivery points under that subscriber.
+Once a service has more than one push service provider and more than one subscriber, uniqush can push data on behalf of that service to the apps running on subscribers' devices. Server side only needs to provide the service name, subscriber id and data. Uniqush will push the data to all active delivery points under that subscriber.
 
-## Removing Push Service Provider and Unsubscribe
+## Removing a Push Service Provider and Unsubscribing
 
-What I didn't mention is removing a push service provider from a service; and unsubscribing a service. It won't be hard once you understand adding push service provider and subscribing a service. They are just reverse operations.
+What I didn't mention is removing a push service provider from a service and unsubscribing from a service. It won't be hard once you understand adding push service provider and subscribing a service. They are just reverse operations.
 
-## Summery
+## Summary
 
-In general, uniqush provide the following five operations:
+In general, uniqush provides the following seven operations:
 
     Add / Remove a push service provider into/from a service
-    Subscribe / Unsubscribe a service
-    Push data 
+    Add / Remove subscriptions from a subscriber
```

Comment:
Awkward phrasing.

Add / Remove a push service provider to/from a service
Add / Remove a subscription to/from a subscriber